effectful/src/Effectful/Console/ByteString.hs

```diff
@@ -0,0 +1,60 @@
+-- | Lifted functions from "Data.ByteString" that are related to standard
+-- streams.
+-- Like the original module, you probably want to import this module qualified
+-- to avoid name clashes with the functions provided by "Prelude", e.g.:
+--
+-- > import Data.ByteString (ByteString)
+-- > import qualified Data.ByteString as BS
+-- > import Effectful.Console.ByteString (Console)
+-- > import qualified Effectful.Console.ByteString as Console
+--
+module Effectful.Console.ByteString
+  ( -- * Effect
+    Console
+
+    -- ** Handlers
+  , runConsole
+
+    -- * Operations
+  , getLine
+  , getContents
+  , putStr
+  , putStrLn
+  , interact
+  ) where
+
+import Prelude hiding
+  ( getContents
+  , getLine
+  , interact
+  , putStr
+  , putStrLn
+  )
+
+import Data.ByteString (ByteString)
+import qualified Data.ByteString as BS
+import qualified Data.ByteString.Char8 as BS8
+
+import Effectful
+import Effectful.Console.Effect
+import Effectful.Dispatch.Static
+
+-- | Lifted 'BS.getLine'.
+getLine :: Console :> es => Eff es ByteString
+getLine = unsafeEff_ BS.getLine
+
+-- | Lifted 'BS.getContents'.
+getContents :: Console :> es => Eff es ByteString
+getContents = unsafeEff_ BS.getContents
+
+-- | Lifted 'BS.putStr'.
+putStr :: Console :> es => ByteString -> Eff es ()
+putStr = unsafeEff_ . BS.putStr
+
+-- | Lifted 'BS8.putStrLn'.
```

Comment:
I think you can `import qualified Data.ByteString.Char8 as BS` and then all functions can uniformly use the `.Char8` variant, right?

Reply:
Done.